frontend/src/metabase/nav/components/search/SearchResults/SearchResults.tsx

```diff
@@ -15,14 +14,25 @@ import { useSearchListQuery } from "metabase/common/hooks";
 import { useDispatch } from "metabase/lib/redux";
 import Search from "metabase/entities/search";
 import { Loader, Text, Stack } from "metabase/ui";
-import type { CollectionItem, SearchModelType } from "metabase-types/api";
+import type {
+  SearchResults as SearchResultsType,
+  CollectionItem,
+  SearchModelType,
+} from "metabase-types/api";
+import {
+  EmptyStateContainer,
+  SearchResultsList,
+} from "metabase/nav/components/search/SearchResults/SearchResults.styled";
 
 type SearchResultsProps = {
   onEntitySelect?: (result: any) => void;
   forceEntitySelect?: boolean;
   searchText?: string;
   searchFilters?: SearchFilters;
   models?: SearchModelType[];
+  footer?:
```

Comment:
Probably a nit but from the name I was expecting this to accept jsx, it instead is a render function/component. Not sure if we have a standard for naming for this case, I see that mantine uses `xxxComponent` in [Select](https://mantine.dev/core/select/#custom-item-component)

Reply:
I'll use `Component` for now, but maybe it's a conversation we should bring up in the next FE meeting